boards/px4/fmu-v5/init/rc.board_sensors

```diff
@@ -26,4 +29,6 @@ ist8310 -C -b 2 start
 ist8310 -C -b 5 start
 
 # Possible pmw3901 optical flow sensor
-pmw3901 start
+# pmw3901 start
```

Comment:
Need to figure out way to coexist.

Reply:
Probably parameters to explicitly enable.